internal/pkg/skuba/kubeadm/configmap_test.go

```diff
@@ -353,6 +353,19 @@ func TestUpdateClusterConfigurationWithClusterVersion(t *testing.T) {
 			currentAdmissionPlugins:  []string{},
 			expectedAdmissionPlugins: []string{"NamespaceLifecycle", "LimitRanger", "ServiceAccount", "TaintNodesByCondition", "Priority", "DefaultTolerationSeconds", "DefaultStorageClass", "PersistentVolumeClaimResize", "MutatingAdmissionWebhook", "ValidatingAdmissionWebhook", "ResourceQuota", "StorageObjectInUseProtection", "RuntimeClass", "NodeRestriction", "PodSecurityPolicy"},
 		},
+		{
+			name:                    "1.18.5 without duplicates",
+			clusterVersion:          version.MustParseSemantic("1.18.5"),
+			currentAdmissionPlugins: []string{},
+			expectedAdmissionPlugins: []string{
+				"NamespaceLifecycle", "LimitRanger", "ServiceAccount", "TaintNodesByCondition",
+				"Priority", "DefaultTolerationSeconds", "DefaultStorageClass",
+				"PersistentVolumeClaimResize", "MutatingAdmissionWebhook",
+				"ValidatingAdmissionWebhook", "ResourceQuota",
+				"StorageObjectInUseProtection", "RuntimeClass",
+				"CertificateApproval", "CertificateSigning", "CertificateSubjectRestriction",
+				"DefaultIngressClass", "NodeRestriction", "PodSecurityPolicy"},
+		},
 	}
```

Comment:
This needs to be rebased and adjusted to include `ExtendedResourceToleration` https://github.com/SUSE/skuba/pull/1192

Reply:
https://github.com/SUSE/caasp-rfc/pull/62

Reply:
It has that already.

Reply:
Rebased against 1.18.6, includes ExtendedResourceToleration now